addons/mail/static/src/models/channel_member_list_view.js

```diff
@@ -1,66 +1,73 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
-import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ChannelMemberListView',
     identifyingFields: [['chatWindowOwner', 'threadViewOwner']],
+    lifecycleHooks: {
+        _created() {
+            this.channel.fetchChannelMembers();
+        },
+    },
     recordMethods: {
+        /**
+         * Handles click on the "load more members" button.
+         */
+        async onClickLoadMoreMembers() {
+            this.channel.fetchChannelMembers();
+        },
         /**
          * @private
-         * @returns {FieldCommand}
+         * @returns {Boolean}
          */
-        _computeChannel() {
-            if (this.chatWindowOwner) {
-                return replace(this.chatWindowOwner.thread);
+        _computeAllMemberLoaded() {
+            if (!this.channel) {
+                return;
             }
-            if (this.threadViewOwner) {
-                return replace(this.threadViewOwner.thread);
-            }
-            return clear();
+            return this.channel.memberCount < this.channel.channelMembers.length;
         },
         /**
          * @private
          * @returns {FieldCommand}
          */
-        _computeOfflineCategoryView() {
-            if (this.channel && this.channel.orderedOfflineMembers.length > 0) {
-                return insertAndReplace();
+        _computeChannel() {
+            if (this.chatWindowOwner) {
+                return replace(this.chatWindowOwner.thread.channel);
+            }
+            if (this.threadViewOwner) {
+                return replace(this.threadViewOwner.thread.channel);
             }
             return clear();
         },
         /**
          * @private
-         * @returns {FieldCommand}
+         * @returns {Boolean}
          */
-        _computeOnlineCategoryView() {
-            if (this.channel && this.channel.orderedOnlineMembers.length > 0) {
-                return insertAndReplace();
+        _computeHasMembers() {
+            if (!this.channel) {
+                return;
```

Comment:
Shouldn't you return `clear()`?

Reply:
I don't know, the purpose is to do nothing. `clear()` does something I guess.

Reply:
Ping @alexkuhn

Reply:
No `return;` in compute methods. It should always return a proper resulting value to the field.
Here it's `return clear()` because it's not applicable when there is no channel, so you simply rely on default value.